examples/step-1/doc/intro.dox

```diff
@@ -63,7 +63,7 @@ sections:
        interpretation. This section also frequently has a subsection
        that gives suggestions on how to extend the program in various
        direction; in the earlier programs, this is intended to give
-       you directions for little experiments designed to make your
+       you directions for little experiments designed to make you
```

Comment:
Thank you for taking the time to also fix these little issues -- the edits are much appreciated!